theSrc/scripts/Quadrants.js

```diff
@@ -120,6 +121,114 @@ function drawQuadrantColors (layout, config, ranges) {
         })
     }
 }
+function drawQuadrantTitles (layout, config, ranges, state) {
+    const { x_min, x_max, y_min, y_max } = ranges
+    if (!layout.annotations) layout.annotations = []
+    let index = layout.annotations.length
+    if (config.quadrantTopLeftTitle && config.yMidpoint < y_max &&
+        config.xMidpoint > x_min) {
+        const initial_offset = state.isStoredInState('quadrantTitle' + index)
+            ? state.getStored('quadrantTitle' + index)
+            : { ax: 0, ay: 0 }
+        layout.annotations.push({
+            text: config.quadrantTopLeftTitle,
+            xanchor: 'left',
+            xref: 'x',
+            ax: initial_offset.ax,
+            x: x_min,
+            xshift: 50,
+            yanchor: 'top',
+            yref: 'y',
+            ay: initial_offset.ay,
+            y: y_max,
+            yshift: -50,
```

Comment:
I ended up positioning the titles at the 4 corners instead of the top middle because it looked better that way. Most examples online put it there as well.

Reply:
Maybe but this is different from the spec. We might have to expose options to let the user choose the position (dragging won't be precise enough).

Reply:
Maybe. Let's see what Tim says tomorrow